tests/unit_tests/synthetic/libraries/cutensor2hiptensor.cu

```diff
@@ -69,4 +68,15 @@ int main() {
   cutensorStatus_t TENSOR_STATUS_INSUFFICIENT_WORKSPACE = CUTENSOR_STATUS_INSUFFICIENT_WORKSPACE;
   cutensorStatus_t TENSOR_STATUS_INSUFFICIENT_DRIVER = CUTENSOR_STATUS_INSUFFICIENT_DRIVER;
   cutensorStatus_t TENSOR_STATUS_IO_ERROR = CUTENSOR_STATUS_IO_ERROR;
+
+  // CHECK: hiptensorAlgo_t tensorAlgo_t;
+  // CHECK-NEXT hiptensorAlgo_t TENSOR_ALGO_DEFAULT = HIPTENSOR_ALGO_DEFAULT;
+  cutensorAlgo_t tensorAlgo_t;
+  cutensorAlgo_t TENSOR_ALGO_DEFAULT = CUTENSOR_ALGO_DEFAULT;
+#endif
+
+#if CUTENSOR_VERSION >= 1400
```

Comment:
`CUTENSOR_VERSION` appeared in cuTensor 1.3.2 and can't be used in the test dedicated to all cuTensor versions

Reply:
Okay let me add it to a separate test.

Reply:
> Okay let me add it to a separate test.

Better to do it in the same test file, but using `CUTENSOR_MAJOR`, `CUTENSOR_MINOR`, and `CUTENSOR_PATCH` defines.

Reply:
> > Okay let me add it to a separate test.
> 
> Better to do it in the same test file, but using `CUTENSOR_MAJOR`, `CUTENSOR_MINOR`, and `CUTENSOR_PATCH` defines.

Sure..